populate_ci_postgres_db.js

```diff
@@ -0,0 +1,29 @@
+const { Client } = require('pg');
```

Comment:
@tgsoverly where you think the right folder is to store this script?

Reply:
I would put it in the test folder, maybe `test/setup`?